webpack/assets/javascripts/react_app/redux/middlewares/APIMiddleware.js

```diff
@@ -0,0 +1,34 @@
+import API from '../../API';
+import { API_OPERATIONS } from '../consts';
+import actionTypeGenerator from '../actionTypeGenerator';
+
+const get = async (payload, url, store, actionTypes) => {
```

Comment:
How can you test it if it doesn't get exported?
I would suggest moving it to a separate file with a separate unit-test.
We are going to end up with more operations such as `put, post, delete, ...`

Reply:
I think most operations will be in the same function because the handling is the same for each one, so they should be in the same file. I'll add the function tests